packages/react-bindings/src/styles/calculateAnimationTimeout.ts

```diff
@@ -0,0 +1,10 @@
+import convertCssTimeToNumber from './convertCssTimeToNumber'
+import * as _ from 'lodash'
+
+const calculateAnimationTimeout = (duration: string, delay: string | undefined): number => {
+  if (_.isNil(duration) || duration === 'inherit' || duration === 'initial') return 0
+  if (!_.isNil(delay) && (delay === 'inherit' || delay === 'initial')) return 0
```

Comment:
```suggestion
  if (delay === 'inherit' || delay === 'initial') return 0
```

I don't think that `isNil()` is a real part of condition there